src/Components/Classes/index.jsx

```diff
@@ -141,61 +67,37 @@ const Classes = () => {
                     <td className={styles.medium}>{theOne?.trainerId?.firstName}</td>
                     <td className={styles.small}>{theOne?.slots}</td>
                     <td className={styles.small}>
-                      <button
-                        onClick={() => {
-                          setCurrentId(theOne?._id);
-                          setShowEdit(!showEdit);
-                          setShowAdd(false);
-                        }}
-                      >
-                        Edit Item
-                      </button>
+                      <Link to={`/classes/form/${theOne?._id}`}>
+                        <Button text={'Edit Item'} variant={'edit'} />
+                      </Link>
                     </td>
                     <td>
-                      <button
-                        className="deleteButton"
-                        onClick={() => {
-                          // eslint-disable-next-line no-restricted-globals
-                          if (confirm('Are you sure?') === true) {
-                            deleteClass(theOne?._id);
-                          }
+                      {showModal && (
```

Comment:
The delete modal isn't working, when you press the delete icon nothing happens, this is because you have to pass the isOpen and onClose properties to the modal.